<launch>
      <arg name="controller_settings_file_path" default="settings.cfg"/>
      <arg name="mode" default="keyboard"/>
      <arg name="joy_path" default="/dev/input/js0"/>

      <node name="interface" pkg="mid_level_controller" type="teleop_car" args="--cfg $(arg controller_settings_file_path) --mode $(arg mode) --joy_path $(arg joy_path)" required="true" output="screen" >
      </node>
</launch>
